instrumentation/github.com/emicklei/go-restful/otelrestful/internal/gen.go

```diff
@@ -0,0 +1,18 @@
+// Copyright The OpenTelemetry Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package internal // import "go.opentelemetry.io/contrib/instrumentation/github.com/emicklei/go-restful/otelrestful/internal"
+
+// Generate semconvutil package:
+//go:generate cp -rf ../../../../../../internal/shared/semconvutil .
```

Comment:
I decided to propose using `cp -rf` instead of `gotmpl` becasue:

1. I find it easier to operate on Go files than on Go template files (we can run `go` commands, run linters, etc)
2. `gotmpl` would need to be invoked on all template files separately (5 lines of `//go:generate` instead of one)
3. I `cp -rf` is simpler `than `gotmpl` 😉 

The downside of using `cp -rf` is that `make vanity-import-fix` needs to be run after `make generate` to fix the vanity import paths.

Reply:
The `cp -rf` approach would not help if there would be dependencies between multiple internal packages. For instance `"go.opentelemetry.io/otel/exporters/otlp/otlpmetric/internal/oconf"` depends on `"go.opentelemetry.io/otel/exporters/otlp/internal/envconfig"`. I am refactoring to use `gotmpl`.

Reply:
+1 to `gotmpl`

Reply:
SIG meeting: We agreed to use `gotmpl`

Reply:
Done https://github.com/open-telemetry/opentelemetry-go-contrib/pull/3817/commits/decfa0da1449339bf6054c6891fa6c0109a529c6